grammars/language-asciidoc.cson

```diff
@@ -587,18 +587,22 @@ repository:
   strong:
     patterns: [
       {
-        name: "markup.bold.unconstrained.asciidoc"
-        match: "\\\\?(?:\\[([^\\]]+?)])?(\\*\\*)(.+?)(\\*\\*)"
+        name: "markup.bold.strong.asciidoc"
```

Comment:
I see your comment on https://github.com/asciidoctor/atom-language-asciidoc/issues/53#issuecomment-217783602

I prefer keep `unconstrained` and `constrained` because if I disable a rule, I don't show properly the edge effects.